### DISCUSSION TITLE
Welcome to AYCE & Eaternity Poster discussions!

### DISCUSSION THREAD

**Discussion by mklarmann** [AYCE-earth/public#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OOAM** [comment]
Hi there.
Very nice poster - congrats!

I would have some questions about the content and the calculations. On which unit are those numbers calculated (per kg, per meal, etc.)? Are these calculations referred to Switzerland?

There are some surprising result: for example how good Kangaroo meat is... Isn't there a lot of transport from Australia to Switzerland?

Best regards

**Reply by mklarmann** [reply]
Hi OOam

There is indeed some documentation missing. That is why we created this forum. For your information the reference is documented here: https://eaternity.org/blog/Eaternity-Rating-Update - but we will have to be more detailed.

Cheers,
Manuel
